### DISCUSSION TITLE
Wechsel des HM-MOD-RPI-PCB bei RaspberryMatic HA (Supervised-Modus)

### DISCUSSION THREAD

**Discussion by estegewr** [OpenCCU/OpenCCU#2315] [🇩🇪 Fragen&Antworten]
Beim Wechsel von FHEM auf HA mit RaspberryMatic habe ich die Konfiguration erst mit einem Testsystem aufgebaut und die HM-Geräte erfolgreich angelernt. Dabei habe ich einen normalen HM-MOD-RPI-PCB als RF-Sender/Empfänger genutzt.

Für das Produktiv-System wollte ich nun die Sendeeinheit ersetzen und einen anderen HM-MOD-RPI-PCB, jedoch mit [externer Antenne](https://www.stall.biz/produkt/komplettbausatz-dualbeam-antenne/) nutzen. Dieser gatte zuvor zuvor unter FHEM zuverlässig funktioniert. Die RF-Einheit war also bereits in einer anderen Umgebung im Einsatz.

RaspberryMatic findet diesen RF-Hardware beim Booten jedoch nicht (klarer Hinweis im Boot-Log).

Ist der adhoc-Wechsel der Sendeeinheit bei Raspberry-Matic problematisch? Oder muss ich die Fehlerursache bei meiner lokalen Installation fortsetzen.

**Comment by jens-maus** [comment]
Nein, ein Wechsel des Funkmodules sollte auf Hardwareebene keinerlei Probleme darstellen und das andere Funkmodul sollte beim hochfahren genauso erkannt werden wie das jeweils alte. 

Die Frage wäre also was konkret im boot.log zu finden ist und welche zugrundeliegende Hardware/System als Host für die HA Supervised Installation genutzt wird. Auch könntest du direkt innerhalb von RM schauen warum das Funkmodul nicht erkannt wird und ggf sogar `detect_radio_module` dort direkt aufrufen.
